runelite-client/src/main/java/net/runelite/client/plugins/objectindicators/ObjectIndicatorsConfig.java

```diff
@@ -94,18 +94,31 @@ default boolean highlightTile()
 	@Alpha
 	@ConfigItem(
 		position = 4,
-		keyName = "markerColor",
-		name = "Marker color",
-		description = "Configures the color of object marker",
+		keyName = "borderColor",
```

Comment:
key name changes force resets the setting for everyone. Don't change the key name.